src/components/Toaster/index.js

```diff
@@ -5,11 +5,9 @@ import {
   ToastContainer as BaseToastContainer,
   toast as baseToast,
 } from 'react-toastify'
-import { injectStyle } from 'react-toastify/dist/inject-style'
 import Icon from '../Icon'
 import NotificationBar from '../NotificationBar'
-
-injectStyle()
+import 'react-toastify/dist/ReactToastify.css'
```

Comment:
Does this correctly inline the css in the bundle ?

Reply:
Do I need to do the same as you did with PhoneInput css ?

Reply:
If this is not inlined yes I think you'll need to

But what's wrong with the injectStyle way ?

Reply:
injectStyles use document object under the hood which is not supported in SSR/SSG mode. The other way to keep it is to put injectStyles() in useLayoutEffect/useEffect on mount lifecycle

Reply:
injectStyle use document object under the hook (See https://github.com/fkhadra/style2js/blob/main/src/style2js.ts). NextJS doesn't like it 😅

Reply:
I just checked and css seems to be correctly inlined in the bundle, so no issue on my side :)